src/Microsoft.Azure.SignalR/HubHost/ServiceLifetimeManagerBase.cs

```diff
@@ -298,6 +397,17 @@ protected static bool IsInvalidArgument(IReadOnlyList<object> list)
         protected ReadOnlyMemory<byte> SerializeProtocol(string protocol, string method, object[] args) =>
             _messageSerializer.SerializeMessage(protocol, new InvocationMessage(method, args));
 
+        protected IDictionary<string, ReadOnlyMemory<byte>> SerializeCompletionMessage(CompletionMessage message)
```

Comment:
No need to return dictionary, return `ReadOnlyMemory<byte>` directly?

Reply:
fixed in #1684